implementation/observation-otel-bridge/src/main/java/io/smallrye/opentelemetry/instrumentation/observation/cdi/ObservationRegistryProducer.java

```diff
@@ -41,9 +39,16 @@ public ObservationRegistry registry() {
                                 openTelemetry.getPropagators().getTextMapPropagator()),
                         new PropagatingReceiverTracingObservationHandler(tracer,
                                 openTelemetry.getPropagators().getTextMapPropagator()),
-                        //   new TracingAwareMeterObservationHandler(tracer) // For exemplars... Maybe not be needed
-                        openTelemetryObservationHandler))
-                .observationHandler(new DefaultMeterObservationHandler(registry));
+                        new OpenTelemetryObservationHandler(tracer)))
+                // todo duplicate the tracer strategy for baggage, adding a condition to bypass when no baggage is present
+                // todo just implement the receiver and sender handlers
+                // todo. Alternatively we can split in 2 the tracing handlers, one to create spans (in the current .observationHandler(new ObservationHandler.FirstMatchingCompositeObservationHandler )
+                // todo. A new .observationHandler bloc to process the baggage on the receiver side.
+                // todo. Another to propagate the context in a new .observationHandler )
+                // todo. We assume on the receiver side we open and close the baggage once because it should have just 1 scope app wide and the
+                // todo. user will use the baggage api itself. We are just making sure we don't break the propagation to the user.
```

Comment:
Is this ok?

Reply:
It means it doesn't provide baggage propagation through the Observation API. The OTel API can be used instead... 
Implementing this is not trivial and the comment explains the possible approach.